src/routes/Outcome.tsx

```diff
@@ -35,7 +35,9 @@ const Outcome = () => {
                     outcome === undefined
                       ? OUTCOME_ROUTE.GENERIC_ERROR
                       : outcome,
-                    walletId?.value
+                    outcome !== undefined && outcome === 0
```

Comment:
```suggestion
                    outcome === 0
```

This should be sufficient, right?

Reply:
I'm not so sure. What's typescript behavior if I try to check `undefined` with `0`?

Reply:
=== 0 should be sufficient :D (just tested it with playground)